### DISCUSSION TITLE
AVLN2002 Duplicate x:Class directive, (My App).App is already used in /App.axaml

### DISCUSSION THREAD

**Discussion by JAAYapps** [AvaloniaUI/Avalonia#17478] [Q&A]
### Describe the bug

I'm having an issue with the axaml files for some reason. Everything in the file is correct in my App.axaml but it's giving me this AVLN2002 Duplicate x:Class directive, {My app name here}.App is already used in /App.axaml

I'm not sure what is going on and I looked everywhere for a solution online. I can compile the app on Linux just fine, but not Windows 10 or 11. I'm not sure what is happening because I made another test project and it works on Windows and Linux. 

![Screenshot_20241111_154759](https://github.com/user-attachments/assets/8b4277ca-91c9-4b03-bea5-c1c6db2c636d)

![Screenshot_20241111_154633](https://github.com/user-attachments/assets/8aae46e9-cc35-4ed2-8177-abc9dbcff695)

### To Reproduce

Unsure how to reproduce due to difficulties trying to create issue on other Avalonia projects.

### Expected behavior

Should just compile without issue on Windows and Linux.

### Avalonia version

11.2.0

### OS

Windows

### Additional context

I can't reproduce the issue on new projects. Sorry if it's not clear. I tried downgrading the Avalonia version, and changing the x:Class to an invalid namespace (Didn't give a different error, but uses the invalid namespace name in the same error).

**Comment by maxkatz6** [comment]
There likely is some misconfiguration in your old project. Can't tell what's exactly without project itself.
Could be duplicated registrations in csproj file.

**Comment by jurihock** [comment]
Hi! I had exactly the same build error. In my case, however, this happened because I accidentally backed up the project into the project folder itself. As a result, the project files were effectively duplicated.

**Comment by NeverMorewd** [comment]
I got the same error and fixed it finally.I would like to share my workaround here:

Remove the following node in the project file:

```xml
  <ItemGroup>
    <AvaloniaResource Include="**\*.axaml" />
  </ItemGroup>
```

**Reply by NeverMorewd** [reply]
By the way...it was caused by llm

**Reply by ENjoyBlue2021** [reply]
> By the way...it was caused by llm

Funny, I had the exact same problem and response. Thought I would use the new Opus 4.7 to guide me through a Avalonia Project. Bad idea apparently. Appreciate the info.
